lib/drivers/native/config_darwin.go

```diff
@@ -1,3 +1,5 @@
+//go:build darwin
```

Comment:
Copyright should be at the top when possible, so better to move it after the copyright - those will work anywhere in the code like here:
https://github.com/adobe/aquarium-fish/blob/eb5358df5a278369dece44b63d30b92528d7cb18/lib/openapi/openapi.go#L10-L16